tracer/resources/applications.py

```diff
@@ -116,7 +116,10 @@ def _append_application(default_attrs, specific_attrs={}):
 	@staticmethod
 	def _helper(app):
 		if app.type == Applications.TYPES["DAEMON"]:
-			return "service {0} restart".format(app.name)
+			if System.init_system() == "systemd":
```

Comment:
Aha, in this case feel free to use whatever you want on Arch :-). But not generally for systemd please, cause on Fedora we use systemd and `service` is prefered.